### ISSUE TITLE
Research synthesis and debrief drafts

### ISSUE THREAD

**Issue by dylansmith-Nava** [CDCgov/prime-reportstream#11376]
## User Story
As a ReportStream teammate, I want to synthesize the various Sender and STLTs meeting notes to date, in order to share out helpful insights, pain points, and opportunity areas with the larger team about improving the onboarding experience.

## Background and context
Part of improving the onboarding experience involves observing users as they navigate the various touchpoints and tasks of the onboarding process. By watching how users interact with the materials, tools, and other documentation that they have access to, we can understand the gaps and frustrations that may occur as users attempt to onboard successfully. Likewise, noting the interactions between users and RS admins can shine a helpful light on how to improve the process for RS admins as well.

## Working links
- [MedArbor Weekly Sync notes](https://docs.google.com/document/d/1xQPASiXAT_HuFfUWf3I1-rTjzbbUgd9jU8DBxPzLT6I/edit)
- [MedArbor Research Synthesis Debrief](https://docs.google.com/document/d/1uZAvTmeMSRv0VydA7B5fHYbXaVoKoz5AgYjMEQjfqek/edit#heading=h.wrj0hrdzckg6)
- [Arkansas Research Synthesis Debrief] (https://docs.google.com/document/d/1VRMdTwUZRKYt1waaeSaV4NuiJWB8FXSbWUSH4wE3JhQ/edit#heading=h.ght1j66yl23v)
- [Arkansas weekly sync] (https://docs.google.com/document/d/19Uh1rKZoigWiN8py_6K3DuRttkQnp5dt_UZjF2zhHwU/edit#heading=h.uraqtbv0n4vp)
- [Previous ticket - Shadow sprint: continuous discovery design brief #9799] (https://app.zenhub.com/workspaces/engagement-6166edbd409257001e09f1b2/issues/gh/cdcgov/prime-reportstream/9799)

## Acceptance criteria
- [x] Write out MedArbor synthesis and debrief draft
- [x] Write out Arkansas synthesis and debrief draft
- [X] Share out with team
- [ ] Finalize after comments

## Not in scope
- Comprehensive solutioning
- Immediate applications for other Senders

**Comment by dylansmith-Nava** [comment]
Finished draft of MedArbor research [here](https://docs.google.com/document/d/1uZAvTmeMSRv0VydA7B5fHYbXaVoKoz5AgYjMEQjfqek/edit#heading=h.deuwfyiblnq5). Need to share with the team. CC @chinelo-ikejimba

**Comment by chinelo-ikejimba** [comment]
Thanks for tagging me @dylansmith-Nava. I'll review the draft on Tues (9/26) before EOD.

**Comment by dylansmith-Nava** [comment]
Finished draft of Arkansas research [here](https://docs.google.com/document/d/1VRMdTwUZRKYt1waaeSaV4NuiJWB8FXSbWUSH4wE3JhQ/edit). Shared with team and waiting on feedback. CC @jillian-hammer

**Comment by dylansmith-Nava** [comment]
Met with Jillian and have a few more questions for Arkansas.

- What follow-up materials were sent to AR after kick-off in April?
- How long did JG spend on making the custom SOAP transport (more than a REST setup)?

**Comment by dylansmith-Nava** [comment]
Finalized the last two questions about with responses from the onboarding team! The answers can be found in the [Arkansas synthesis](https://docs.google.com/document/d/1VRMdTwUZRKYt1waaeSaV4NuiJWB8FXSbWUSH4wE3JhQ/edit) document. As such, I am closing out this ticket. CC @jillian-hammer, looks like the design work has been completed here. Also CC @chris-kuryak.
